app/views/tags/tags/_primary_publishing_organisation.html.erb

```diff
@@ -0,0 +1,9 @@
+<% if edition.access_limit.present? %>
```

Comment:
We should call the directory `fields/`, since they're intended to form part of the wider body of field (that just happen to be shown only on this page at the moment).

Reply:
Can I just confirm you'd like this PR to implement this structure?

```diff
  views/
    content/
-       edit/
-           _body_field.html.erb
     edit.html.erb
+   fields/
+       edit/
+           _body_field.html.erb
+           _primary_publishing_organisation_field.html.erb
+       show/
+           _body_field.html.erb
+           _primary_publishing_organisation_field.html.erb
    documents/
        show/
-          contents/
-              _body_field.html.erb
        _actions.html.erb
        ...etc
    tags/
-      tags/
+      types/
            _multi_tag_input.html.erb
            _single_tag_input.html.erb
     edit_api_down.html.erb
     edit.html.erb
```

I.e. consolidate content and tag fields under one `fields/` directory, with their 'edit' view under a `edit/` subdirectory and their rendered view under a `show/` subdirectory? And `fields/edit/_primary_publishing_organisation_field.html.erb` referencing the `tags/types/_single_tag_input.html.erb` partial?

Reply:
Thanks for elaborating - it's good to be clear 👍. I'd say "yes", except that I don't see the need for `_single_tag_input.html.erb`, if it's only used once - why have indirection without motivation?

Reply:
Agreed 👍